server/src/main/java/org/apache/druid/client/TimelineServerView.java

```diff
@@ -36,6 +38,9 @@
   @Nullable
   TimelineLookup<String, ServerSelector> getTimeline(DataSource dataSource);
 
+  @Nullable
```

Comment:
Does this need to be nullable? None of the implementations return null.

Reply:
Right, they throw UnsupportedOperationException instead, removed the nullable annotation.